### PR TITLE
updated the code systems and their official names

### LINKED ISSUE #101: CodySystems Names
We should have consistency in the `name` of all ontologies used. 

e.g. ORDO should be:
```
ORDO = CodeSystem(
    name="Orphanet Rare Disease Ontology",
    namespace_prefix="ORPHA",
    url="http://www.orpha.net/"
```

we can take the official names and official namespace_prefixes from OLS: 
https://www.ebi.ac.uk/ols4/ontologies

### PR DESCRIPTION
**Pull request by aslgraefe**
added the code systems, edited the official names, and added versions

### CODE REVIEW

File: src/phenopacket_mapper/data_standards/code_system.py [hunk 1]
```diff
@@ -49,67 +49,128 @@ def __contains__(self, item):
...
-    name="Human Phenotype Ontology",
-    namespace_prefix="HP",
-    url="http://www.human-phenotype-ontology.org",
-    iri_prefix="http://purl.obolibrary.org/obo/HP_",
-    synonyms=["HPO"]
+
+ICD11 = CodeSystem(
+    name='International Classification of Diseases, Eleventh Revision', 
+    namespace_prefix='icd11', 
+    url='https://icd.who.int/en'
```
Comment (frehburg) [inline src/phenopacket_mapper/data_standards/code_system.py]:
```suggestion
    url='https://icd.who.int/en'
    synonyms=['ICD-11']
```

File: src/phenopacket_mapper/data_standards/code_system.py [hunk 2]
```diff
@@ -49,67 +49,128 @@ def __contains__(self, item):
...
+    namespace_prefix='ICD10GM', 
+    url='https://www.bfarm.de/EN/Code-systems/Classifications/ICD/ICD-10-GM/_node.html#:~:text=ICD%20%2D10%2D%20GM%20is%20an,means%20%22German%20Modification%22).',
+    synonyms=['ICD10-GM']
+)
+
+ICD10CM = CodeSystem(
+    name='International Classification of Diseases, Tenth Revision, Clinical Modification', 
+    namespace_prefix='ICD10CM', 
+    url='https://www.cdc.gov/nchs/icd/icd10cm.htm',
+    synonyms=['ICD10-CM', 'ICD10_CM']
```
Comment (frehburg) [inline src/phenopacket_mapper/data_standards/code_system.py]:
```suggestion
    synonyms=['ICD10-CM', 'ICD10_CM', 'ICD-10-CM', 'ICD_10_CM']
```

File: src/phenopacket_mapper/data_standards/code_system.py [hunk 3]
```diff
@@ -49,67 +49,128 @@ def __contains__(self, item):
...
+    namespace_prefix='ICD9',
+    url='https://www.cdc.gov/nchs/icd/icd9.htm',
+    synonyms=['ICD-9']
+)
+
+ICD10_GM = CodeSystem(
+    name='International Classification of Diseases, Tenth Revision, German Modification', 
+    namespace_prefix='ICD10GM', 
+    url='https://www.bfarm.de/EN/Code-systems/Classifications/ICD/ICD-10-GM/_node.html#:~:text=ICD%20%2D10%2D%20GM%20is%20an,means%20%22German%20Modification%22).',
+    synonyms=['ICD10-GM']
```
Comment (frehburg) [inline src/phenopacket_mapper/data_standards/code_system.py]:
```suggestion
    synonyms=['ICD10-GM', 'ICD-10-GM', 'ICD10_GM', 'ICD_10_GM']
```

File: src/phenopacket_mapper/data_standards/code_system.py [hunk 4]
```diff
@@ -49,67 +49,128 @@ def __contains__(self, item):
...
-    iri_prefix="http://purl.obolibrary.org/obo/HP_",
-    synonyms=["HPO"]
+
+ICD11 = CodeSystem(
+    name='International Classification of Diseases, Eleventh Revision', 
+    namespace_prefix='icd11', 
+    url='https://icd.who.int/en'
+)
+
+HL7FHIR = CodeSystem(
```
Comment (frehburg) [inline src/phenopacket_mapper/data_standards/code_system.py:105]: Is FHIR a code system?

File: src/phenopacket_mapper/data_standards/code_system.py [hunk 5]
```diff
@@ -49,67 +49,128 @@ def __contains__(self, item):
...
 )
 MONDO = CodeSystem(
     name="Monarch Disease Ontology",
     namespace_prefix="MONDO",
     url="http://purl.obolibrary.org/obo/mondo.owl"
 )
+ORDO = CodeSystem(
+    name="Orphanet Rare Disease Ontology",
+    namespace_prefix="ORPHA",
+    url="http://www.orpha.net/"
```
Comment (frehburg) [inline src/phenopacket_mapper/data_standards/code_system.py]:
```suggestion
    url="http://www.orpha.net/"
    synonyms=['ORDO']
```

File: tests/utils/parsing/test_parse_data_type.py
Comment (frehburg) [inline tests/utils/parsing/test_parse_data_type.py:1]: Please don't change the test cases. Re-import ICD9 and leave tests as they were before

File: tests/utils/parsing/test_parse_value_set.py
Comment (frehburg) [inline tests/utils/parsing/test_parse_value_set.py:1]: Please don't change the test cases. Re-import ICD9 and leave tests as they were before

File: src/phenopacket_mapper/data_standards/code_system.py
Comment (frehburg) [inline src/phenopacket_mapper/data_standards/code_system.py:1]: a few synonyms were missing but it looks good otherwise

**Review by frehburg — CHANGES_REQUESTED** [review]
Please return tests to their previous state and review the synonym additions

File: src/phenopacket_mapper/data_standards/code_system.py (hunk 4 above)
Comment (aslgraefe) [reply src/phenopacket_mapper/data_standards/code_system.py:105]: yes

File: tests/utils/parsing/test_parse_value_set.py
Comment (aslgraefe) [reply tests/utils/parsing/test_parse_value_set.py:1]: done

File: src/phenopacket_mapper/data_standards/code_system.py
Comment (aslgraefe) [reply src/phenopacket_mapper/data_standards/code_system.py:1]: ty

File: src/phenopacket_mapper/data_standards/code_system.py [hunk 6]
```diff
@@ -49,67 +49,128 @@ def __contains__(self, item):
...
+    name='Sequence types and features ontology', 
+    namespace_prefix='SO',
+    url='http://www.sequenceontology.org/'
+)
+
+ICD9 = CodeSystem(
+    name='International Classification of Diseases, Ninth Revision',
+    namespace_prefix='ICD9',
+    url='https://www.cdc.gov/nchs/icd/icd9.htm',
+    synonyms=['ICD-9', 'ICD_9', 'icd9']
```
Comment (frehburg) [inline src/phenopacket_mapper/data_standards/code_system.py]:
```suggestion
    synonyms=['ICD-9', 'ICD_9']
```

case is ignored when matching

**Review by frehburg — APPROVED** [review]

**Pull request merged**